api/web/templates/web/project_view.html

```diff
@@ -383,15 +400,34 @@ <h3 class="mb-0">
         data: data,
         colHeaders: JSON.parse("{{columns|escapejs}}"),
         rowHeaders: true,
+        height: 620,
         filters: true,
         dropdownMenu: true,
-        viewportRowRenderingOffset: 100,
+        viewportRowRenderingOffset: 50,
         stretchH: 'all',
         licenseKey: 'non-commercial-and-evaluation',
     });
     Handsontable.dom.addEvent(window, 'hashchange', function (event) {
         hot.loadData(getData());
     });
 
+    var button1 = document.getElementById('export-file');
+    var exportPlugin1 = hot.getPlugin('exportFile');
+
```

Comment:
Adicionei o listener dos eventos "show.bs.collapse" e "shown.bs.collapse".
O evento "show.bs.collapse" é disparado no momento em que o collapse é aberto.
O evento "show**n**.bs.collapse" é disparado no momento em que o conteúdo collapse é totalmente renderizado.

Dessa forma, um spinner vai ser apresentado para o usuário ao abrir o collapse e o mesmo spinner vai ser removido quando o html da tabela for renderizada e em seguida, é executada a função "hot.render();" para a tabela ser carregada pelo script do handsontable.

```suggestion

    $("#collapseTwo").on("show.bs.collapse", function(){
        $("#data-table-loading").show();
        $("#data-table").hide();
    });

    $("#collapseTwo").on("shown.bs.collapse", function(){
        $("#data-table-loading").hide();
        $("#data-table").show();
        hot.render();
    });

```